src/pylexibank/metadata.py

```diff
@@ -1,12 +1,40 @@
+import re
 import pkg_resources
 
 import attr
 from cldfbench.metadata import Metadata
 
-__all__ = ['LexibankMetadata']
+__all__ = ['LexibankMetadata', 'check_standard_title']
 
 version = pkg_resources.get_distribution('pylexibank').version
 
+STANDARD_TITLE_PATTERN = re.compile(
+    r'CLDF dataset derived from\s+'
+    r'(?P<authors>[^"]+)'
+    r'"(?P<title>[^"]+)"\s+from\s+'
+    r'(?P<year>[0-9]{4})'
+)
+
+
+def check_standard_title(title):
+    """
+    Assert a title conforms to the standard format.
+
+    > CLDF dataset derived from AUTHOR's "TITLE" from YEAR
+
+    Usage: In a dataset's `test.py`:
+    ```python
+    from pylexibank import check_standard_title
+
+    def test_valid_title(cldf_dataset, cldf_logger):
+        check_standard_title(cldf_dataset.metadata_dict['dc:title'])
+    ```
+
+    Note that this requires installing `pylexibank` for tests, also in .travis.yml.
+    """
+    match = STANDARD_TITLE_PATTERN.fullmatch(title)
+    assert match and match.group('authors').strip().endswith("'s")
```

Comment:
This fails if the name ends in "s" and we should only have an apostrophe (e.g., `Thomas'`).

Reply:
Yes, for this reason, I added the line:
```
assert match.group('authors').strip().endswith("'s") or match.group('authors').strip().endswith("s'")
```

Reply:
but there were too many threads where I posted this.

Reply:
I am not about the rule that sibilants should have an -es. Like `of Finch` becoming `Finches` instead of `Finch's` (or just `Finch'`)?

Reply:
I missed it, sorry.

Reply:
BTW, `endswith` can take a tuple if you think it is more legible, so `.endswith(("'s", "s'"))`